pkgs/by-name/as/asfa/package.nix

```diff
@@ -0,0 +1,63 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  stdenv,
+  darwin,
+  help2man,
+  openssl,
+  pkg-config,
+}:
+let
+  version = "0.10.0";
+in
+rustPlatform.buildRustPackage {
+  pname = "asfa";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "obreitwi";
+    repo = "asfa";
+    rev = "v${version}";
+    hash = "sha256-MnhnwtZmPFhOuiqNiaxJnPu88JOdlpvyVy0YGphblBc=";
+  };
+
+  cargoHash = "sha256-/bRBP/NzcNOXl/nANeOYouUAo3NNbtbV9fxIJrNajYQ=";
+
+  buildInputs = [
+    openssl
+  ] ++ lib.optionals stdenv.isDarwin [ darwin.apple_sdk.frameworks.SystemConfiguration ];
+
+  nativeBuildInputs = [
+    help2man
+    pkg-config
+  ];
+
+  doCheck = false;
```

Comment:
```suggestion
  # <explain why doCheck is false>
  doCheck = false;
```

Reply:
NixOS provides a test framework that fires QEMU virtual machines. But we can do this another day.

Reply:
Yeah, been meaning to look into that. In other projects, I would like to run integration tests against ephemeral databases in pure nix derivations. Currently, those are set up by running docker images.